docs/integrations/Trust-any-certificate.md

```diff
@@ -0,0 +1,101 @@
+# Trust Any Certificate
+
+## Overview
+
+Disabling SSL verification by setting `verify=False` in `BaseClient` (or by enabling “Trust Any Certificate” in an integration) allows connecting to servers using less-secure SSL configurations. This bypasses standard HTTPS checks—skipping certificate validation, lowering OpenSSL’s security level, and permitting older ciphers and renegotiation modes.
+
+## Usage
+
+In your integration’s when constructing `BaseClient`, set `verify=False` to disable SSL checks and allow legacy ciphers:
```

Comment:
When constructing the `BaseClient` in your integrations, set `verify=False` to disable SSL checks and allow legacy ciphers: